src/components/Profile/Profile.tsx

```diff
@@ -1,69 +1,169 @@
-import React, { useState } from "react";
-// import {updateReport} from "../../helpers/apiCalls"//iTHink this has to be pass down as a prop from app
+import React, { useState, useEffect, useContext, useRef} from "react";
 import bkg_img from "../../images/report/Big Shoes - Jumping On One leg Pose.png";
 import "./Profile.css";
+import { updateUserReport } from "../../helpers/apiCalls";
+import { ReportContext, AnswerContext } from "../../helpers/context";
+import { Answers } from "../../helpers/types";
 interface profileProp {
   updateReport: any;
 }
 
 export const Profile: React.FC<profileProp> = ({ updateReport }) => {
-  const [profileInfo, updateProfileInfo] = useState<any>({})
+  
+  const report = useContext(ReportContext)
+  const answers = useContext(AnswerContext);
+  const [error, setError] = useState<string>('');
+  const [person, setPerson] = useState<string>('');
+  const [email, setEmail] = useState<string>('');
+  const [st, setAllAnswers] = useState<any>(answers);
+
+  useEffect(() => {
+    const userError = 'no name found'
+    const emailError = 'no email found'
+    const user = localStorage.getItem('displayName') || userError;
+    const email = localStorage.getItem('email') || emailError;
+    setPerson(user)
+    setEmail(email)
+  }, [])
+
+  const cleanUp =(newReport: any) => {
+    const formattedAnswers: Answers = {
+      salary: newReport.monthly_salary,
+      zipcode: newReport.zip_code,
+      credit_score: newReport.credit_score,
+      monthly_debt: newReport.monthly_debt,
+      downpayment_savings: newReport.downpayment_savings,
+      mortgage_term: '30',
+      downpayment_percentage: newReport.downpayment_percentage,
```

Comment:
I wonder if we need to send mortgage term now that it is hard-coded on the backend